olad/www/new/index.html

```diff
@@ -7,6 +7,39 @@
   <title>OLA</title>
   <link rel="stylesheet" href="/new/libs/bootstrap/css/bootstrap.min.css"/>
   <link rel="stylesheet" href="/new/css/style.min.css"/>
+  <link rel="icon" type="image/png" sizes="192x192" href="img/favicons/android-chrome-192x192.png">
```

Comment:
I think judging on what we've done elsewhere in the file, these may need to be fully qualified with /new/ on the start. It would be good to be consistent anyway.